crates/api_models/src/payment_methods.rs

```diff
@@ -144,6 +144,17 @@ pub struct PaymentMethodResponse {
     pub created: Option<time::PrimitiveDateTime>,
 }
 
+#[derive(Clone, Debug, Eq, PartialEq, serde::Deserialize, serde::Serialize)]
+pub struct CardDetailsPaymentMethod {
+    pub scheme: Option<String>,
+    pub last4_digits: Option<String>,
+    pub issuer_country: Option<String>,
+    pub expiry_month: Option<masking::Secret<String>>,
+    pub expiry_year: Option<masking::Secret<String>>,
+    pub nick_name: Option<masking::Secret<String>>,
+    pub card_holder_name: Option<masking::Secret<String>>,
```

Comment:
I see that `PaymentMethod` struct already contains `scheme`, `issuer_country` and `card_holder_name`. can we ignore those fields here?

Reply:
Believe we don't store issuer_country and card_holder_name as such in the table. The fields are always empty. So we can get those fields from locker response. Have made changes for scheme.